spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-sample/src/main/java/com/example/WebController.java

```diff
@@ -58,8 +60,8 @@ public RedirectView createSubscription(@RequestParam("topicName") String topicNa
 
 	@GetMapping("/postMessage")
 	public RedirectView publish(@RequestParam("message") String message,
-			@RequestParam("topicName") String topicName) {
-		this.pubSubTemplate.publish(topicName, message, null);
+			@RequestParam("topicName") String topicName) throws IOException {
```

Comment:
Seems for me we have just talked that there is no `IOException` requirement any more. 
Let's not overcomplicate samples!